### ISSUE TITLE
.DS_Store and *~ file in RC release tarball

### ISSUE THREAD

**Issue by proppy** [GoogleCloudPlatform/kubernetes#3359]
I noticed that the [0.8 RC](https://github.com/GoogleCloudPlatform/kubernetes/releases/tag/v0.8.0) has some temporary file in the binary tarball.

```
examples/.DS_STORE
cluster/get-kube.sh~
...
```

It would be nice to add release validation script, so that those don't show up in the official release.

**Comment by brendandburns** [comment]
Fixed by #4074

**Comment by grigio** [comment]
I get this error in a Linux host and vagrant, maybe it's releated to this
```
rm: cannot remove ‘/home/grigio/Code/kubernetes/_output/release-stage/full/kubernetes/examples/rethinkdb/.DS_Store’: No such file or directory
```
